src/Symfony/Component/Uid/AbstractUid.php

```diff
@@ -11,10 +11,15 @@
 
 namespace Symfony\Component\Uid;
 
+use Ds\Hashable;
+
+// ensure we have _some_ version of the Ds\Hashable interface
+require_once __DIR__.'/DsHashablePolyfill.php';
```

Comment:
We rely on the autoloader for loading code. No `require_once` please.

Reply:
usually we do, but there are places in the Symfony codebase which do explicit `require`s and I thought this would qualify.. but the alternate approach suggested by @nicolas-grekas doesn't even need that, so it should be better now.